styles/styles.css

```diff
@@ -578,9 +578,9 @@ button {
 }
 
 .button-container a.tertiary {
-  height: 20px;
-  line-height: 20px;
+  line-height: var(--spacer-element-06);
   align-items: end;
+  text-decoration: none;
```

Comment:
I removed this by accident and have added it back in my branch. It actually needs to be added to `.button-container a`